screen_layout

```diff
@@ -0,0 +1,35 @@
+# the following two lines give a two-line status, with the current window highlighted
```

Comment:
Please leave editor preferences out of the repository.

You can use them and keep them untracked by adding them to the global gitignore or the local ignore in `.git/info/exclude`.